plugin/pkg/skplug/proto/skplug.proto

```diff
@@ -48,17 +48,35 @@ message StatRequest {
   repeated Stat stat = 2;
 }
 
-message ScaleRequest {
+
+message VerticalSizeRequest{
+string partition = 1;
+    int64 time_nanos = 2;
+    }
+
+message VerticalSizeResponse{
```

Comment:
Nit: let's keep the method name consistent.  Change to `VerticalRecommendationResponse`.

Reply:
Fixed it. See commit ff6c618226dcbefd5bb69d64b65764a9f71a2834.